Formula/k/kafka.rb

```diff
@@ -112,7 +112,7 @@ def install
       system "#{bin}/kafka-topics --bootstrap-server localhost:#{kafka_port} --delete --topic test " \
              ">> #{testpath}/kafka/demo.out 2>/dev/null"
     ensure
-      system "#{bin}/kafka-server-stop"
+      system bin/"kafka-server-stop"
       system "#{bin}/zookeeper-server-stop"
```

Comment:
you missed the others

Reply:
@issyl0 Any idea why autocorrect didn't catch these?

Reply:
The original spec for this audit was "#{bin}/<formula>-and-subcommands", so that's what I did. We can iterate in future though!